pkg/common/common.go

```diff
@@ -134,6 +136,10 @@ const (
 	CollectionDiskQuotaKey       = "collection.diskProtection.diskQuota.mb"
 
 	PartitionDiskQuotaKey = "partition.diskProtection.diskQuota.mb"
+
+	// database level properties
+	DatabaseReplicaNumber  = "database.replica.number"
+	DatabaseResourceGroups = "database.resource_groups"
```

Comment:
Should we make a default database config to make different replica in different resource group?
Current implementation doesn't support it.